to-do-list.css

```diff
@@ -0,0 +1,66 @@
+html,
+body {
+  font-family: 'VT323', monospace;
+  /* font-family: 'Teko', sans-serif; */
+  font-size: 25px;
+  background-color: #0a766f;
+  color: #a6fe01;
+  display: flex;
+  flex-direction: column;
+  margin: 0;
+  padding:0;
+}
+.big-header {
+  padding-left: 15px;
+  padding-bottom: 0;
+  margin: 0;
+}
+
+.label {
+  padding-left: 15px;
+}
+input[type="text"], textarea {
+  padding: 0;
+  background-color: #0ea49a;
+  border-style: dashed;
+  border-radius: 5px;
+  border-width: 1px;
+  border-color: #a6fe01;
+  color: #a6fe01; /* green */
+  /* color: #ff8000; orange */
+  font-family: 'VT323', monospace;
+  font-size: 20px;
+}
+
+.button {
+  font-family: 'VT323', monospace;
+  font-size: 20px;
+  color: #a6fe01;
+  padding: 2.5px;
+  border-radius: 5px;
+  background-color: #0ea49a;
+  border-style: solid;
+  border-color: #a6fe01;
+}
+
+.button:link,
+.button:visited {
+  background-color: #5c14eb;
+  color: #4d4d4d;
+  border-style: dashed;
+  border-color: #4d4d4d;
+}
+.button:hover,
+.button:visited:hover{
+  color: #0ea49a;
+  background-color: #a6fe01;
+  border-color: #0ea49a;
+
+}
+.button:active,
+.button:visited:active {
+  color: #ff8000;
+  background-color: #bd0000;
+  border-color: #ff8000;
+
```

Comment:
✂️

Reply:
Same as above. Should I cut it? If so, why?